src/oas-parsing/operation-example/operation-example.factory.ts

```diff
@@ -9,13 +9,14 @@ export default class OperationExampleFactory {
 
     for (const operation of operations) {
       const exampleGroups = operation.exampleGroups;
+      const exampleRequestBodies = operation.exampleRequestBodies;
 
       for (const exampleGroup of exampleGroups) {
-        operationExamples.push({
-          operation,
-          exampleGroup,
-          requestBody: operation.exampleRequestBody,
-        });
+        for (const exampleRequestBody of exampleRequestBodies) {
```

Comment:
I like for loops more than most but three nested in one method calls for it to be a new function

Reply:
Refactored this logic